casa/Quanta/MVTime.h

```diff
@@ -460,9 +460,19 @@ ostream &operator>>(ostream &is, MVTime &meas);
 ostream &operator<<(ostream &os, const MVTime::Format &form);
 // </group>
 
-// is equal operator, uses operator Double which returns days
+// equality and comparisson operators, use operator Double which returns days
```

Comment:
```suggestion
// equality and comparison operators, use operator Double which returns days
```

Reply:
Changed, I always get confused with this one...